langserve/server.py

```diff
@@ -333,6 +335,52 @@ def _get_base_run_id_as_str(
         raise AssertionError("No run_id found for the given run")
 
 
+def _json_encode_response(obj: BaseModel, *, is_batch: bool = False) -> JSONResponse:
+    """Return a JSONResponse with the given content."""
+    obj = jsonable_encoder(obj)
+
+    if is_batch:
+        # Handle Batch Response
+        if not isinstance(obj["output"], list):
+            raise AssertionError("Expected output to be a list")
+
+        # Collapse '__root__' from output field if it exists. This is done
```

Comment:
qq for my understanding: what does this __root__ mean?

Reply:
Pydantic convention for validating someting like:

[1, 2, 3]

```
class SomeModel(BaseModel):
   __root__: List[int]
```

In pydantic 2 this is ```RootModel[List[int]]```